### DISCUSSION TITLE
some extensions don't work

### DISCUSSION THREAD

**Discussion by kuzivas** [lllyasviel/stable-diffusion-webui-forge#127] [Q&A]
Hi! For me, Forge generates about 20% faster than A1111 (RTX A4000). But Deforum, ReActor, FaceFusion do not work. FaceSwaplab doesn't save images in the output. I haven't tested anything else yet. I was thinking of switching to this UI, but now I don't know. Will this be fixed?

**Comment by deagzzz** [comment]
Someone said it's based on dev build of A1111 so that's why some extensions are broken. I hope it get's fixed soon as I'm also waiting for civitai helper to work and then i'll make a switch

**Comment by lllyasviel** [comment]
hi please do not spread misinformation

reactor works well without problems.

facefusion is a software that is not a webui extension

deforum is heavily changing webui behaviors and usually deforum users use independent webui install to just use deforum, even when using standard webui 1.7.0

**Reply by dhmiller123** [reply]
Love your work. I am a heavy (exclusive) Deforum user and would love to see support for it to gain the clear speed advantages (4600) I'm seeing with i2i.  Hope you can find time, and thanks again for your work.
